### ISSUE TITLE
Homework2 parseMessage does invalid parsing

### ISSUE THREAD

**Issue by clrowd** [bschwb/cis194-solutions#6]
Passing strings like "E is an unknown string" (which should be parsed as Unknown String) to `parseMessage` will make it match the Error type, and results in 
`LogMessage (Error *** Exception: Prelude.read: no parse`.

**Comment by goforit2017** [comment]
Yes, that is a problem this solution didn't cover. I think the message must in a specific style like if the first char is "E" then the following message is supposed to be "70 3 Way too many pickles" to show the level of severity. Hope this helps.
